### PR TITLE
add more ICEs

### PR DESCRIPTION
**Pull request by matthiaskrgr**

### CODE REVIEW

File: ices/91801-2.rs [hunk 1]
```diff
@@ -0,0 +1,5 @@
+trait Foo<'a> {}
+
+fn or<'a>(first: &'static dyn Foo<'a>) -> dyn Foo<'a> {
+    return Box::new(panic!());
+}
```
Comment (Alexendoo) [inline ices/91801-2.rs:5]: Can remove this file since it's https://github.com/rust-lang/rust/issues/91803

**Review by Alexendoo — COMMENTED** [review]
Thanks! nit: could you also rename 91801.rs/91827.rs to also have hyphens in them

**Comment by matthiaskrgr** [comment]
done, renamed to `-0.rs`

**Review by Alexendoo — APPROVED** [review]
Thanks!

**Pull request merged**